docs/source/usecases/s3encryption.rst

```diff
@@ -14,6 +17,59 @@ remediate them such that they are encrypted.
            crypto: aws:kms
 
 Options
-  ``crypto`` for determining the crypto mechanism, this can either be ``aws:kms`` or ``AES256`` (default)
-  ``key-id`` for specifiying the customer KMS key to use for the SSE, if the ``crypto`` value passed is ``aws:kms`` the AWS default KMS key will be used instead.
-  ``report-only`` generate reports of unencrypted keys in a bucket, but do not remediate them.
++++++++
+
+  - ``crypto`` for determining the crypto mechanism, this can either be
+    ``aws:kms`` or ``AES256`` (default)
+
+  - ``key-id`` for specifiying the customer KMS key to use for the SSE, if the
+    ``crypto`` value passed is ``aws:kms`` the AWS default KMS key will be used
+    instead.
+
+  - ``report-only`` generate reports of unencrypted keys in a bucket, but do
+    not remediate them.
+
+
+Remediate Incoming
+------------------
+
+Will scan all newly created objects and remediate them such that they are
+encrypted.
+
+.. code-block:: yaml
+
+   policies:
+     - name: s3-attach-encryption
+       resource: s3
+       actions:
+         - type: attach-encrypt
+           role: arn:aws:iam::123456789012:role/my-role
```

Comment:
the role aspect seems to be confusing people wrt to the same doc item used in config-rule. note the role is optional if folks are using --assume-role on the command-line. worth noting.